lib/src/components/VernaToolbar/index.tsx

```diff
@@ -3,10 +3,12 @@ import ShowCaseWidgetProps from ':/types/Widgets';
 
 interface VernaToolbarProps {
   children: ReactElement[];
+  onDragging: (isDragging: boolean) => void;
```

Comment:
Why not just use callback `onDragStart` and `onDragEnd` ? It's more event compliant than use one callback triggered when drag state change